fb-webworker.js

```diff
@@ -0,0 +1,109 @@
+importScripts('https://cdn.jsdelivr.net/pyodide/v0.23.4/full/pyodide.js');
+
+const EXCLUDE_CHECKS = [
+  'com.google.fonts/check/fontbakery_version', // We download the latest each time
```

Comment:
> "We download the latest each time"

each day the web version is updated. But I can imagine that in the future we may forget to keep it updated and it would be useful for users to know that it is not really the very latest available version.

But maybe this check also has trouble due to needing to access the network for querying pypi...

Reply:
> No, fontbakery-web literally calls `pip install fontbakery` every time you _open the web page_. So it always gets the latest (pre-release) from pypi.

Awesome!